cs/troubleshooting.texy

```diff
@@ -81,3 +81,30 @@ Pokud je nutné vypsat tyto znaky v situaci, kdy by se text chápal jako značka
 Chybí mi knihovna pro AMF, Amazon S3, OpenID, …
 -----------------------------------------------
 Nette Framework nechce být rozsáhlou knihovnou řešící úplně vše, ale frameworkem, tedy uceleným souborem přicházejícím s určitým stylem práce (tzv. Nette-way). Framework je proto relativně malý, ale existuje pro něj řada [doplňků |https://componette.org], jejichž nejucelenější sadou je pak Symfony.
+
+
+
+Nette generuje odkazy s HTTP misto HTTPS
+-----------------------------------------------
+Pokud provozuje Nette aplikaci za proxy Nette vám vrací URL s HTTP místo HTTPS, je problém s nastaveném serveru a/nebo Nette.
+
+Pokud používáte jako proxy Nginx je potřeba mít nastaveno přesměrování např. takto:
+
+```
+        location / {
+                proxy_set_header Host $host;
+                proxy_set_header X-Real-IP $remote_addr;
+                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+                proxy_set_header X-Forwarded-Proto $scheme;
+                proxy_set_header X-Forwarded-Port  $server_port;
+                proxy_set_header X-Forwarded-Host  $host;
+                proxy_pass http://IP-aplikace:80;
+        }
+```
+
+Dále je potřeba do Neon souboru doplňit tyto řádky:
+
+```
+http:
+	proxy: IP/IProzsah-proxy
+```
```

Comment:
Šlo by to ještě víc pro blbý? :-) Těmto věcem nerozumím a tady bych hodně tápal v tom kde vezmu IP a hlavně ten rozsah. (Nemluvě o lidech, kteří by přesně tahle to zkopírovali do konfiguráku a divili by se, proč to nefunguje).

Reply:
Aktualne nevim jak vic to rozepsat pro mene znale a nebyl cas se vzit do kuze druhe strany a promyslet to. Co by se ti libilo tam mit napsane?

Reply:
V těchto věcech se vůbec nevyznám, takže vlastně tester toho, jestli je to napsané pro blbé. Vím teda to, že do konfigurace se nemá napsat "IP/IProzsah-proxy", ale místo toho něco jiného. To by tam mělo být jasně řečeno. Ale já už nevím, co bych ta měl napsat a kde ty čísla vezmu.